readme.md

```diff
@@ -115,14 +119,14 @@ Write some basic CSS that demonstrates changing a CSS property when the device w
 How is rebase different than a merge?
 
 Your Answer:
-```text
-
+```
+	It seems as if merging is better because when you merge, you still have history of that branch.  But when you rebase, it overwrites and fully commits everything onto the master branch. The benefit of rebase is that it gives you a cleaner project history.
 ```
 
 ### Question #10
 
 Describe some of the common git workflows for teams (fork and pull request, etc).
 
-```text
-
+```
+	fork, git clone, git push origin [branch], git pull, git merge, and merge conflicts. 
 ```
```

Comment:
There are plenty of workflows a team can use:

1. Everyone is a contributor, and simply pushes their work to master.
2. Everyone is a contributor, but pushes their work to feature branches. Other team members merge in the branches at their discretion. Pull Requests may or may not be used.
3. Everyone forks the repo, and creates Pull Requests from their fork to the master. One or more people are in charge of reviewing and approving Pull Requests.

In addition to the above, teams may set their own policies on code review, when PRs can be merged, etc.

Additionally, teams may decide on a convention for naming their branches, and how changes get merged into the master branch (e.g. there may be an intermediate Development branch for integrating multiple PRs and testing how they work together.) One such example is the git flow model.